crates/goose-server/src/commands/agent.rs

```diff
@@ -15,8 +18,19 @@ pub async fn run() -> Result<()> {
     let secret_key =
         std::env::var("GOOSE_SERVER__SECRET_KEY").unwrap_or_else(|_| "test".to_string());
 
-    // Create app state - agent will start as None
-    let state = state::AppState::new(secret_key.clone()).await?;
+    // TODO: Create agent using provider and model as args to CLI command or environment variables
```

Comment:
i think we'll need to add CLI params to `goosed` here for initial model and provider